tests/console/validate_zfcp_multipath.pm

```diff
@@ -23,31 +23,20 @@ sub verify_host_bus_adapters_attached {
     }
 }
 
-# Verify that there are 8 SCSI block devices listed (as a result of having two adapters, two hard disks and two paths to each disk, 2^3 = 8)
-# Verify that LUN 0x4001403200000000 corresponds to the 214GB disk, and LUN 0x4001405000000000 to the 42.9GB disk
```

Comment:
# Verify that there are 2 SCSI block devices listed
# Verify that LUN 0x0000000000000000 corresponds to 53.6GB disk

Update the description like this?

Reply:
Updated comments